cli.js

```diff
@@ -9,6 +9,12 @@ const index = require('./')
 const pkg = require('./package.json')
 
 const client = new AWS.Kinesis()
+const zlib = require('zlib')
+
+const gzip_head = Buffer.from([0x1f, 0x8b])
```

Comment:
How about casing these like `GZIP_HEAD` so they look more like global constants instead of regular variables?